ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -15,6 +15,8 @@ spec:
     - name: pyxis_url
       default: https://pyxis.engineering.redhat.com
   results:
+    - name: update_cert_project_status
+      description: The boolean value indicating if the cert project status needs to be updated or not
```

Comment:
What about changing the result name/description for `certification_status`? It's misleading right now because I'd expect that to return the current status rather than a _possible_ future status. It would be useful to return both current and future states to clear up any potential confusion.